examples/UKCore-Immunization-Sn-PatientConformVaccination-Example.xml

```diff
@@ -13,7 +13,7 @@
 	</vaccineCode>
 	<!--  **************snippet start**************  -->
 	<patient>
-		<reference value="urn:uuid:edea022a-2d81-11eb-adc1-0242ac120002"/>
+		<reference value="Patient/UKCore-Patient-RichardSmith-Example"/>
```

Comment:
do we need a reference AND an identifier?

Reply:
Hi @APageNHS thanks for looking at this.  Although we have an identifier for an NHS number, the reference is to a particular instance of a patient example.  So we keep both, although I see what you mean.